pkg/cluster/spec/validate.go

```diff
@@ -307,6 +307,64 @@ Please change to use another port or another host.
 	return nil
 }
 
+// TiKVLabelError indicates that some TiKV servers don't have correct labels
+type TiKVLabelError struct {
+	TiKVInstances map[string][]error
+}
+
+// Error implements error
+func (e *TiKVLabelError) Error() string {
+	str := ""
+	for id, errs := range e.TiKVInstances {
+		if len(errs) == 0 {
+			continue
+		}
+
+		str += fmt.Sprintf("%s:\n", id)
+		for _, e := range errs {
+			str += fmt.Sprintf("\t%s\n", e.Error())
+		}
+	}
+	return str
+}
+
+// CheckTiKVLocationLabels will check if tikv missing label or have wrong label
+func CheckTiKVLocationLabels(pdLocLabels []string, kvs []TiKVSpec) error {
+	lerr := &TiKVLabelError{
+		TiKVInstances: make(map[string][]error),
+	}
+	lbs := set.NewStringSet(pdLocLabels...)
```

Comment:
It's better to check whether the `pdLocLabels` is empty, so does `kvs`.

Reply:
1. If the `kvs` labels is empty and two kv deployed to the same host, it will report an error
![image](https://user-images.githubusercontent.com/8718109/93880790-9b001780-fd10-11ea-830f-783ecf612d81.png)
2.If the `kvs` labels is empty but the `pdLocLabels` is, it will report the label is not found in PD config
![image](https://user-images.githubusercontent.com/8718109/93880846-ac492400-fd10-11ea-8cf2-262f8b862d18.png)

So the empty `pdLocLabels` is not a validate case.